syncosaurus/syncosaurus.js

```diff
@@ -1,6 +1,7 @@
 import { WriteTransaction, ReadTransaction } from './transactions';
-import { authHandler } from './authHandler.js';
-const roomUriPrefix = 'ws://localhost:8787/websocket/room';
+
+// these need to be moved with the next commit
+const roomUriPrefix = 'ws://localhost:8787';
```

Comment:
I think this will be resolved when you pull changes from main because this will be an environment variable